crates/swc_ecma_transforms_module/tests/.old.amd.rs

```diff
@@ -0,0 +1,1341 @@
+use swc_common::{chain, Mark};
```

Comment:
Can you delete these files?

Reply:
I will delete it after all test cases are migrated.